EventFilter/L1TRawToDigi/plugins/implementations_stage2/EMTFBlockSP.cc

```diff
@@ -16,6 +16,13 @@ namespace l1t {
         bool unpack(const Block& block,
                     UnpackerCollections* coll) override;  // Apparently it's always good to use override in C++
         // virtual bool packBlock(const Block& block, UnpackerCollections *coll) override;
+
+      private:
+        bool useHMTBits_{false};
+        bool useNNBits_{false};
```

Comment:
```suggestion
```
No need to have the two bools here as class members: define them as local variables inside the `unpack` method, instead